R/bs-dependencies.R

```diff
@@ -48,18 +36,18 @@
 #' }
 #'
 #' # Latest bootstrap
-#' preview_button(bootstrap())
+#' preview_button(bs_dependencies())
 #' # Bootstrap 3
-#' preview_button(bootstrap("3"))
+#' preview_button(bs_dependencies("3"))
 #' # Bootswatch minty theme
-#' preview_button(bootstrap("minty"))
+#' preview_button(bs_dependencies("minty"))
 #' # Bootswatch sketchy theme
-#' preview_button(bootstrap("sketchy"))
+#' preview_button(bs_dependencies("sketchy"))
 #' # Bootswatch solar theme with BS3 compatibility
-#' preview_button(bootstrap("solar@4+3"))
+#' preview_button(bs_dependencies("solar@4+3"))
 #'
-bootstrap <- function(
-  theme = bs_theme_get(),
+bs_dependencies <- function(
+  theme = bs_global_get(),
```

Comment:
Maybe this function shouldn't have a default for `theme`, but `bootstrap()` could have it default to `bs_global_get()`?

Reply:
Yea, that makes sense, I'll do the same for `bs_sass()`